README.md

```diff
@@ -82,11 +82,17 @@ See the section about [running tests](https://facebook.github.io/create-react-ap
 To run the test runner with emulators, use:
 
 ```bash
-firebase emulators:exec --project sample --only firestore 'npm test'
```

Comment:
Why this change?

Reply:
Whoops, missed this one: basically as an easier copy/paster since we have the --only demonstrated below